s29744-JakubMarkowski/zadania 1-14 30.09.txt

```diff
@@ -0,0 +1,86 @@
+
+Zadania wykonuje za pomocą VirtualBox.
+
+1.Uruchom terminal.
+
+
+1.prawy przycisk myszy, open in terminal.
+2.menu, terminal.
+3.sprawdziłem w internecie, jest możliwość ctrl + alt + T
+
+
+2.Wyświetl zawartość aktualnego katalogu.
+
+ls
+
+Wyświetla zawartość katalogu
+
+3.Sprawdź gdzie aktualnie się znajdujesz.
+
+pwd
+
+Wyświetla aktualną ścieżke
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej
+
+cd Desktop/
+
+5.Sprawdź gdzie jesteś.
+
+pwd
+
+6.Wyświetl zawartość bieżącego katalogu.
+
+ls 
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z odpowiednim parametrem).
+
+ls ~
+
+przechodze do katalogu domowego
+
+8.Przejdź do katalogu Obrazy w twoim katalogu domowym za pomocą ścieżki względnej.
+
+cd ~/Music/
+
+cofam się do katalogu domowego i wchodze do Music
+
+9.Sprawdź gdzie jesteś.
+
+pwd
+
+10.Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././
+W opisie rozwiązania proszę umieścić również to jak rozumiesz co się właściwie stało.
+Dlaczego znalazła/eś się w tym miejscu do którego przeniosła Cię ta ścieżka?
+Czy dodanie kilku ./././././ na początku/środku/końcu powoduje, że ścieżka się zmieni?
+Co ta ścieżka oznacza/sobą reprezentuje?
+
+-u mnie po wpisaniu cd ././././././ nic się nie stało, ponieważ ./ jest moim obecnym katalogiem. Zauważyłem, że polecenie cd ././././home
+przeniesie mnie do home.
+
+11.Przejdź do swojego katalogu domowego za pomocą ścieżki bezwzględnej.
+
+cd /home/jakubmark
+
+12.Zobacz co się stanie jak wpiszesz komendę 'cd /root'. Jak już to zrobisz, zastanów się co się stało. Poszukaj w Internecie.
+
+po wpisaniu komendy, dostałem komunikat o braku dostępu, nie mam uprawnień administratora muszę użyć komendy:
+
+sudo su
+
+13 i 14.
+Zobacz, co się stanie, jeśli wciśniesz kombinację klawiszy: Ctrl+Alt+F3
+Sprawdź w Internecie co się dzieje kiedy używa się takiej kombinacji. W Internecie mogą pojawić się wersje z F1-F6. To de facto to samo.
+Odpowiedź "wylogowało mnie" jest błędna. Nie. Nie wylogowało. Szukaj dalej.
+
+szczerze to musiałem wygooglować bo nie wiedziałem, przenosi nas na 3 konsole bez gui.
```

Comment:
No to jest do przestudiowania. Zagadnienie jest proste ale nie takie łatwe do przedstawienia dlatego trzeba tego doświadczyć empirycznie.